cmd/up/exec_unix.go

```diff
@@ -7,22 +7,13 @@ import (
 	"context"
 	"os"
 	"os/exec"
-	"runtime"
 	"strings"
 	"syscall"
 )
 
+// GetCommandToExec for non-windows GOOS
 func (he *hybridExecutor) GetCommandToExec(ctx context.Context, cmd []string) (*exec.Cmd, error) {
-	var c *exec.Cmd
-	if runtime.GOOS != "windows" {
-		c = exec.Command("bash", "-c", strings.Join(cmd, " "))
-	} else {
-		binary, err := expandExecutableInCurrentDirectory(cmd[0], he.workdir)
-		if err != nil {
-			return nil, err
-		}
-		c = exec.Command(binary, cmd[1:]...)
-	}
+	c := exec.Command("bash", "-c", strings.Join(cmd, " "))
```

Comment:
why are we forcing `bash`?

Reply:
This was already implemented, i've havent changed the code

cc @AdrianPedriza for context

Reply:
It was the same approach used by our default [_executors_](https://github.com/okteto/okteto/blob/master/cmd/utils/executor/executor.go#L61-L64) (run without bash is false by default so in most of the cases we are using `bash`) where we are always using `bash` unless we are running remotely deploy/detroy operation.

Reply:
we agreed to revisit this logic and work on it to unblock all related changes / PRs. This is the new issue: https://github.com/okteto/okteto/issues/3816

Reply:
just for the record, this PR does not change this piece of code, it was a refactor (move of code) that was previously implemented, does this new issue replace this one? i am not understanding.